app/views/capture/index2.html.erb

```diff
@@ -0,0 +1,363 @@
+<%= stylesheet_link_tag "new-capture" %>
+
```

Comment:
Here we can link directly to the relevant JS files from https://github.com/publiclab/spectral-workbench/pull/668 to work outside of the asset pipeline until we get this page working. Can we try that?